CONTRIBUTING.md

```diff
@@ -27,7 +27,7 @@ guidelines:
     only place the text in the GitHub pull request description.
     To amend a missing `CLA: trivial` line after submission, do the following:
 
-  [cla]: https://github.com/ituglib/gmake/policies/cla.html
+  [cla]: policies/cla.html
```

Comment:
hyperlink works, but shows the html source rather then rendering it properly. Convert to markdown?

Reply:
That's weird. I wonder why. Maybe convert to markdown would be good.